spec/serializers/export/own_post_serializer_spec.rb

```diff
@@ -1,12 +1,54 @@
 describe Export::OwnPostSerializer do
-  let(:post) { create(:status_message_with_photo) }
-  subject(:json_output) { Export::OwnPostSerializer.new(post).to_json }
-
-  # TODO: rewrite this spec
-  it { is_expected.to include %("guid":"#{post.guid}") }
-  it { is_expected.to include %("text":"#{post.text}") }
-  it { is_expected.to include %("public":#{post.public}) }
-  it { is_expected.to include %("author":"#{post.diaspora_handle}") }
-  it { is_expected.to include %("entity_type":"#{post.type.underscore}") }
-  it { is_expected.to include %("created_at":"#{post.created_at.iso8601}) }
+  let(:author) { FactoryGirl.create(:user_with_aspect).person }
+
+  before do
+    author.owner.share_with(FactoryGirl.create(:person), author.owner.aspects.first)
+  end
+
+  it_behaves_like "it is federation entity serializer" do
+    let(:object) { create(:status_message_with_photo, author: author) }
+  end
+
+  let(:json) { Export::OwnPostSerializer.new(post, root: false).to_json }
+
+  context "with private post" do
+    let(:post) { create(:status_message_in_aspect, author: author) }
+
+    before do
+      expect(post.subscribers).not_to be_empty
```

Comment:
About false positives: When there are subscribers for one test, then they are also there for the other test, so there is no need to check it on every test. I also don't like `expect` stuff outside of the actual test.

About "tests for test": I think the "better" solution would be, to check the resulting json has the key "subscribed_users_ids" and it isn't empty, in addition to the `include_json` check. That way we would "cleaner" test the result and not the test. But since `expect(post.subscribers).not_to be_empty` in addition with the `include_json` does the same, I think that's also a way to achieve the same result.